ci/playbooks/deploy_podified_ctlplane.yaml

```diff
@@ -0,0 +1,59 @@
+---
+- hosts: controller
+  tasks:
+    - name: Install Ansible, Make, Tar
+      become: true
+      ansible.builtin.package:
+        name:
+          - ansible-core
+          - make
+          - tar
+        state: latest
+
+    - name: Install devsetup dependencies
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/devsetup"
+        target: download_tools
+
+    - name: Deploy Openstack Operators
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+        target: openstack
+        params: &install_yamls_out
+          OUT: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/out"
+          MICROSHIFT: 1
+
+    - name: Wait for operator to get deployed
+      ansible.builtin.shell: |
+        oc get csv --no-headers=true | grep -i "succeeded"
+      register: operator_status
+      until: operator_status.rc == 0
+      retries: 20
+      delay: 5
+
+    - name: Print Operator Status
+      ansible.builtin.shell: |
+        oc get csv;
+        oc get pods;
+
+    - name: Deploy OpenStack Services
+      community.general.make:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
+        target: openstack_deploy
+        params:
+          <<: *install_yamls_out
+          OPENSTACK_REPO: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/openstack-operator"
```

Comment:
thanks for adding it, though this latest version looks a lot different to the previous one, which had a few of these repeating:

OUT: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls/out"

Reply:
marking resolved just a comment. it is fine like this